docs/ERC721.md

```diff
@@ -714,6 +714,9 @@ The ERC165 standard allows smart contracts to exercise [type introspection](http
 
 Cairo contracts, like Ethereum contracts, have no native concept of an interface, so applications must usually simply trust they are not making an incorrect call. For trusted setups this is a non-issue, but often unknown and untrusted third-party addresses need to be interacted with. There may even not be any direct calls to them! (e.g. ERC20 tokens may be sent to a contract that lacks a way to transfer them out of it, locking them forever). In these cases, a contract declaring its interface can be very helpful in preventing errors.
 
+It should be noted that the [constants library](../openzeppelin/utils/constants.cairo) includes constant variables referencing all of the interface ids used in these contracts. This allows for more legible code i.e. `IERC165_ID` vs. `0x01ffc9a7`.
```

Comment:
```suggestion
It should be noted that the [constants library](../openzeppelin/utils/constants.cairo) includes constant variables referencing all of the interface ids used in these contracts. This allows for more legible code i.e. using `IERC165_ID` instead of `0x01ffc9a7`.
```